sysutils/eza/Portfile

```diff
@@ -46,6 +46,9 @@ variant doc description {Build man pages} {
                 -in=${name}.1.md -out=${name}.1"
             system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
                 -in=${name}_colors.5.md -out=${name}_colors.5"
+            system -W ${worksrcpath}/man "${prefix}/bin/go-md2man \
+                -in=${name}_colors-explanation.5.md \
+                -out=${name}_colors-explanation.5"
```

Comment:
I don't think `go-md2man` is needed anymore, since the pandoc port now just [installs the binary](https://github.com/macports/macports-ports/blob/master/textproc/pandoc/Portfile) rather than building from source.